src/components/POLine/POLineForm.js

```diff
@@ -447,6 +450,15 @@ function POLineForm({
                             integrationConfigs={integrationConfigs}
                           />
                         </Accordion>
+                        <Accordion
```

Comment:
Please update the tests for this accordion.

Reply:
updated